src/coreclr/jit/instrsarm64sve.h

```diff
@@ -2758,10 +2716,35 @@ INST1(ftmad,             "ftmad",                 0,                       IF_SV
     // FTMAD   <Zdn>.<T>, <Zdn>.<T>, <Zm>.<T>, #<imm>                                    SVE_HN_2A           01100101xx010iii 100000mmmmmddddd     6510 8000   
 
 
+//    enum               name                     info                                              SVE_HO_3A                                    
+INST1(bfcvt,             "bfcvt",                 0,                       IF_SVE_HO_3A,            0x658AA000                                   )
+    // BFCVT   <Zd>.H, <Pg>/M, <Zn>.S                                                    SVE_HO_3A           0110010110001010 101gggnnnnnddddd     658A A000   
+
+//    enum               name                     info                                              SVE_HO_3B
+INST1(fcvt,              "fcvt",                  0,                       IF_SVE_HO_3B,            0x6508A000)
```

Comment:
bit 23 is always `1` and in the encoding, you are not setting it for all conversions. I am wondering with the current changes, is the encoding matching with capstone?


```suggestion
INST1(fcvt,              "fcvt",                  0,                       IF_SVE_HO_3B,            0x6588A000)
```

Reply:
It is being set but not always explicitly under `(3 << 22)` in some of the cases.